src/components/VrtConversion/Withdraw.tsx

```diff
@@ -37,26 +38,34 @@ export type WithdrawPropsType = {
   handleClickWithdraw: () => void;
 };
 
-export default ({ withdrawableAmount, account, handleClickWithdraw }: WithdrawPropsType) =>
-(
-  <WithdrawWrapper>
-    <div className="withdraw-title">
-      <div className="withdraw-title-line-1">Withdrawable amount</div>
-      <div className="withdraw-title-line-2">
-        {commaFormatter(withdrawableAmount.toFixed(4))}
-        {' '}
-        XVS
+export default ({ withdrawableAmount, account, handleClickWithdraw }: WithdrawPropsType) => {
+  const [withdrawLoading, setWithdrawLoading] = useState(false);
+  return (
+    <WithdrawWrapper>
+      <div className="withdraw-title">
+        <div className="withdraw-title-line-1">Withdrawable amount</div>
+        <div className="withdraw-title-line-2">
+          {commaFormatter(withdrawableAmount.toFixed(6))}
+          {' '}
+          XVS
+        </div>
       </div>
-    </div>
-    <ButtonWrapper>
-      <button
-        type="button"
-        className="button withdraw-button"
-        disabled={!account}
-        onClick={handleClickWithdraw}
-      >
-        {!account ? 'Connect' : 'Withdraw'}
-      </button>
-    </ButtonWrapper>
-  </WithdrawWrapper>
-);
+      <ButtonWrapper>
+        <button
```

Comment:
we already have v/Button component, so you can start using it for new components

Reply:
got it